accord-core/src/test/java/accord/impl/basic/Cluster.java

```diff
@@ -778,7 +780,7 @@ private static void verifyConsistentRestore(Int2ObjectHashMap<NavigableMap<TxnId
                     if (!store.unsafeCommands().containsKey(txnId))
                     {
                         Command beforeCommand = before.get(txnId);
-                        if (beforeCommand.saveStatus() == SaveStatus.Erased)
```

Comment:
Should we permit this? I don't think we should even *have* any commands where `saveStatus` is `Uninitialised` - might indicate some other problem? If the command is `Uninitialised` it means we didn't touch it, so we shouldn't have saved it.

Reply:
Agreed, this should not be the case; fixed now